docs/embedded-server-guide/src/main/asciidoc/embedded-server-guide.adoc

```diff
@@ -1973,40 +1973,40 @@ Line breaks in the following element are for readability purposes only
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.glassfish</groupId>
+    <groupId>org.example</groupId>
     <artifactId>my-ejb-app-tester</artifactId>
-    <version>3.1</version>
-    <name>Maven test</name>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>EJB Embeddable API Example</name>
     <dependencies>
         <dependency>
-            <groupId>org.glassfish.extras</groupId>
+            <groupId>org.glassfish.main.extras</groupId>
             <artifactId>glassfish-embedded-static-shell</artifactId>
```

Comment:
I would replace rhis dependency with the standard GlassFish Embedded dependency: https://central.sonatype.com/artifact/org.glassfish.main.extras/glassfish-embedded-all/

It’s easier to set up and doesn’t require a local GlassFish installation.

Then also remove GLASSFISH_HOME variable and the system scope definition.